client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -45,7 +45,21 @@ class ScanPage extends Component {
 	}
 
 	renderScanning() {
-		return <p>Scanning!</p>;
+		return (
+			<>
+				<SecurityIcon icon="in-progress" />
+				<h1 className="scan__header scan__header--okay">{ translate( 'Preparing to scan' ) }</h1>
+				<ProgressBar value={ 1 } total={ 100 } color="#069E08" />
+				<p>
+					Welcome to Jetpack Scan, we are taking a first look at your site now and the results will
```

Comment:
Is it intentional not to localize this string?

Reply:
I wanted to skip the localization of this for now.